parachains/runtimes/assets/statemine/src/lib.rs

```diff
@@ -82,6 +82,20 @@ impl_opaque_keys! {
 	}
 }
 
+#[cfg(feature = "state-trie-migration")]
+#[sp_version::runtime_version]
+pub const VERSION: RuntimeVersion = RuntimeVersion {
```

Comment:
DQ but why does this introduce a whole new `VERSION`?

Reply:
The sp_version::runtime_version do not accept `cfg(feature...` I remember trying to do the macro change for it in the past, but since this is just rather `temporary` code, having a bit of duplication here looks fine to me.